bower.json

```diff
@@ -18,6 +18,7 @@
     "paper-scroll-header-panel": "~1.0.16",
     "app-layout": "~0.10.2",
     "fecha": "~2.2.0",
-    "paper-range-slider": "IftachSadeh/paper-range-slider#~0.1.2"
+    "paper-range-slider": "IftachSadeh/paper-range-slider#~0.1.2",
+    "micromarkdown.js": "SimonWaldherr/micromarkdown.js#~0.3.0"
```

Comment:
No reason to add this to bower.json since we don't import it. However, please include the version number and license inside the new micromarkdown.js folder in Home Assistant repo.